src/core/annotation.js

```diff
@@ -661,6 +677,63 @@ class WidgetAnnotation extends Annotation {
     return !!(this.data.fieldFlags & flag);
   }
 
+  /**
+   * Parse widget annotation appearance and extract font information.
+   *
+   * @public
+   * @memberof WidgetAnnotation
+   * @param {number} flag - Hexadecimal representation for an annotation
+   *                        field characteristic
+   * @return {Promise}
+   */
+  parseAppearance(params) {
+    if (!params.pdfManager || !params.pdfManager.pdfDocument) {
```

Comment:
This is validation required because of unit-tests. We don't create AcroForm dictionary for them and I tried to minify unit-tests modifications. Normally this validation does nothing, and in this sense it's 100% correct